app/src/main/java/fr/lille1/univ/android/architecture/btrshop/detailsproduct/domain/model/Offer.java

```diff
@@ -0,0 +1,103 @@
+package fr.lille1.univ.android.architecture.btrshop.detailsproduct.domain.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Created by denis on 03/11/2016.
+ */
+
+public class Offer implements Serializable {
```

Comment:
Add a bit of Javadoc on EVERY classes.
+ I'm not 100% convinced by the relevance of this class (but I might be wrong)